### PR TITLE
lesson22-aysel

### PR DESCRIPTION
**Pull request by ayselakpinar**

### CODE REVIEW

File: Lesson22/todo-app/aysel/script.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  const listItem = document.createElement('li');
+  const textSpan = document.createElement('span');
+  textSpan.innerText = input.value;
+  listItem.addEventListener('click', function () {
+    textSpan.classList.toggle('completed');
+  });
+
+  const button = document.createElement('button');
+  button.innerText = 'x';
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/script.js:19]:
```suggestion
  button.innerText = 'x';
  button.classList.add('button--light');
```

Comment (annasvst) [reply Lesson22/todo-app/aysel/script.js:19]: As otherwise styles for the 'x' button are not applied

File: Lesson22/todo-app/aysel/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	<body>
+
+
+		<div class="container">
+
+			<h1>To-Do List</h1>
+
+			<form id="toDoForm">
+				<input type="text" placeholder="Enter to do" required id="toDoInput"/>
+				<button type="submit" >submit</button>
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/index.html]: To style elements, it's always good to add a class:
```suggestion
				<button class="btn" type="submit" >submit</button>
```

File: Lesson22/todo-app/aysel/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+				<button type="reset">reset</button>
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/index.html]:
```suggestion
				<button class="btn" type="reset">reset</button>
```

File: Lesson22/todo-app/aysel/styles.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  
+  input[type="text"] {
+    flex: 1;
+    padding: 10px;
+    font-size: 16px;
+    border: none;
+    border-bottom: 2px solid #ddd;
+  }
+  
+  button[type="submit"] button[type="reset"]{
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/styles.css]: There's a coma (',') missing in this selector definition, should be 'button[type="submit"], button[type="reset"]{' instead but I'd recommend to switch to a class selector:
```suggestion
  .btn {
```

File: Lesson22/todo-app/aysel/styles.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    padding: 12px 24px; 
+  font-size: 16px; 
+  color: white; 
+  border: none; 
+  border-radius: 5px; 
+  cursor: pointer; 
+  transition: background-color 0.3s, color 0.3s; 
+  }
+
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/styles.css:77]: Apart from the styles defined above for the buttons, I'd add different colors to the submit and reset buttons to make the UI more user friendly and increate the contrast between background color of button and it's text (now it's very hard to read):
```
  button[type="submit"] {
    background-color: rgb(43, 77, 70);
  }
  button[type="submit"]:hover {
    color: rgb(43, 77, 70);
    background-color: #fff;
  }
  button[type="reset"] {
    background-color: rgb(109, 41, 41);
  }
  button[type="reset"]:hover {
    color: rgb(109, 41, 41);
    background-color: #fff;
  }
```

File: Lesson22/todo-app/aysel/styles.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  }
+  
+  h1 {
+    text-align: center;
+    color: #333;
+  }
+  
+  form {
+    display: flex;
+    margin-bottom: 20px;
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/styles.css:32]: Add some space between input and buttons:
```suggestion
    margin-bottom: 20px;
    gap: 10px;
```

File: Lesson22/todo-app/aysel/script.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+const form = document.getElementById('toDoForm');
+
+form.addEventListener('submit', handleSubmit);
+form.addEventListener('reset', handleReset);
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/script.js]: Reset button in the form should reset the input element(s). E.g. if there's some data in the input fields in the form, 'reset' button should clear all input fields. 
Current behaviour of resent button is incorrect and would be considered a misleading UI. Instead, remove this line but if you want a user to clear entire todo list by clicking a button, add a new button in HTML outside the form element for that.

File: Lesson22/todo-app/aysel/script.js [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    listItem.remove();
+  });
+
+  listItem.append(textSpan);
+  listItem.append(button);
+  document.getElementById('todoList').append(listItem);
+  
+  input.value = '';
+}
+function handleReset(event) {
```
Comment (annasvst) [inline Lesson22/todo-app/aysel/script.js]: This function is also not needed unless you add a new button in HTML

**Review by annasvst — COMMENTED** [review]
Looks good, approved ✅

**Comment by ayselakpinar** [comment]
> Looks good, approved ✅
:) but i can not merge it unfortunately

**Review by annasvst — APPROVED** [review]

**Pull request merged**